Java/Calculator/Calculator/src/Calculator.java

```diff
@@ -0,0 +1,44 @@
+
+import java.util.Scanner;
+
+public class Calculator {
+    static Scanner sc = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        System.out.println("Calculator");
+        System.out.println("Please enter First number into the console");
+        checkNumberOrString(sc);
+        Double number1 = sc.nextDouble();
+        System.out.println("Please input one the operand as '+ ,- ,* ,/ , % '");
+        char OperationSymbol = sc.next().charAt(0);
+        System.out.println("Please enter Second number into the console");
+        checkNumberOrString(sc);
+        Double number2 = sc.nextDouble();
+        final String SUM = "+";
```

Comment:
constants should be at the beginning of file

Reply:
Done